apps/kg/distributed/partition.sh

```diff
@@ -0,0 +1,17 @@
+##################################################################################
+# User runs this script to partition a graph using METIS
+##################################################################################
```

Comment:
do we have to provide this file?

Reply:
Partition.sh is not only responsible for partition, it will also copy some related files to the partition directory.